robottelo/decorators/host.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
+from functools import wraps
+
+if six.PY3:
+    from functools import lru_cache
+elif six.PY2:
+    # workaround only for this module, its not doing exactly the same as
+    # original from Python 3, but it fits this specific case
+    def lru_cache(maxsize):
+        def decorator(func):
+            cache = []
+
+            @wraps(func)
+            def wrapper():
+                if not cache:
```

Comment:
This is not doing any cache at all since the function will be called every time, I think in this case is better to just create a pass though decorator or implement lru_cache the way it is supposed to work.